spec/lib/zipkin_kafka_tracer_spec.rb

```diff
@@ -92,15 +88,15 @@ def initialize;end
 
       it 'returns if id already sampled' do
         allow(id).to receive(:sampled?) { false }
-        expect(subject).to_not receive(:get_span_for_id)
-        subject.set_rpc_name(id, name)
+        expect(tracer).to_not receive(:get_span_for_id)
+        tracer.set_rpc_name(id, name)
       end
 
       it 'sets the span name' do
         allow(id).to receive(:sampled?) { true }
-        allow(subject).to receive(:get_span_for_id) { span }
+        allow(tracer).to receive(:get_span_for_id) { span }
         expect(span).to receive(:name=).with(name)
-        subject.set_rpc_name(id, name)
+        tracer.set_rpc_name(id, name)
       end
     end
   end
```

Comment:
thanks for getting rid of all those `subject` calls, they didn't make any sense but were out of the scope of my previous PRs so I left them untouched.